src/components/ConfigDropdown.tsx

```diff
@@ -0,0 +1,54 @@
+import './ConfigDropdown.scss'
```

Comment:
What is tsx? A quick Google is only returning cars.

Reply:
Haha. `tsx` is `ts` with JSX support (React). And `ts` is [TypeScript](https://www.typescriptlang.org/).

The idea is that introducing typing is the front-line of defence against introducing bugs.